x-pack/plugins/observability_solution/slo/common/constants.ts

```diff
@@ -87,6 +86,9 @@ export const DEFAULT_SLO_GROUPS_PAGE_SIZE = 25;
 export const getSLOSummaryTransformId = (sloId: string, sloRevision: number) =>
   `slo-summary-${sloId}-${sloRevision}`;
 
+export const getSLOPipelineId = (sloId: string, sloRevision: number) =>
+  `.slo-observability.sli.pipeline-${sloId}-${sloRevision}`;
```

Comment:
``
```suggestion
  `.slo.sli.pipeline-${sloId}-${sloRevision}`;
```

Reply:
why changing the convention used by all other resources for this one?

Reply:
I only suggested in case there is limit, otherwise all is good